lago/cmd.py

```diff
@@ -767,6 +756,20 @@ def check_group_membership():
         warnings.warn('current session does not belong to lago group.')
 
 
+def check_deps():
+    try:
+        # Checks that all the deps are installed
+        pkg_resources.require("lago")[0]
+    except pkg_resources.ContextualVersionConflict as e:
+        # Hack that allows to run stevedore without checking
+        # for it's dep. it is required for systems running stevedore 1.1.0
+        # and pbr > 1.
+        LOGGER.debug(traceback.format_exc())
```

Comment:
`LOGGER.debug(e, exc_info=True)` could be used instead.

Not sure if the message is useful when the exception is re-raised, exception could be logged only when ignored/hidden.